processors/portfolio/portfolio_create-overview.md

```diff
@@ -11,15 +11,16 @@ of this repository.
 | Parameter                     | Required | Description                                                        |
 |-------------------------------|----------|--------------------------------------------------------------------|
 | input.inventory.dir           | yes      | The directory of the input inventory for overview generation.      |
-| input.inventory.path          | yes      | The path of the input inventory for overview generation.           |
-| input.security.policy.file    | yes      | The security policy file used for the overview.                    |     
+| input.inventory.file          | yes      | The file path of the input inventory for overview generation.      |
```

Comment:
input.inventory.subpath; extend comment that this is a relative path from input.inventory.dir